geotrek/feedback/models.py

```diff
@@ -372,9 +385,11 @@ def distance(self, to_cls):
         return settings.REPORT_INTERSECTION_MARGIN
 
     def report_interventions(self):
-        report_content_type = ContentType.objects.get_for_model(Report)
-        qs = Q(target_type=report_content_type, target_id=self.id)
-        return Intervention.objects.existing().filter(qs).distinct('pk')
+        if 'geotrek.maintenance' in settings.INSTALLED_APPS:
+            report_content_type = ContentType.objects.get_for_model(Report)
+            qs = Q(target_type=report_content_type, target_id=self.id)
+            return Intervention.objects.existing().filter(qs).distinct('pk')
```

Comment:
```suggestion
            return Intervention.objects.existing().filter(filters).distinct('pk')
```
qs is queryset synonym